### ISSUE TITLE
Getting overtrained models when training with diffusers 

### ISSUE THREAD

**Issue by killamonis** [huggingface/diffusers#2218]
### Describe the bug

I've been trying to train a model using diffusers with this base model https://huggingface.co/wavymulder/Analog-Diffusion, but I've been getting very overtrained models, even on a low amount of steps. 

These are the parameters I'm passing in 
--pretrained_model_name_or_path="/home/ubuntu/models" 
--instance_data_dir=image path 
--output_dir=output path
--instance_prompt="" 
--seed=random 
--resolution=512
--train_batch_size=1 
--gradient_accumulation_steps=1 
--gradient_checkpointing 
--use_8bit_adam 
--learning_rate=5e-06 
--lr_scheduler="linear" 
--lr_warmup_steps=0 
--max_train_steps=800

I've tried 800 & 500 steps and got very overtrained models
300 or lower gave me bad training results

### Reproduction

Running train_dreambooth.py with the parameters above 

### System Info

- `diffusers` version: 0.13.0.dev0
- Platform: Linux-5.15.0-1021-aws-x86_64-with-glibc2.31
- Python version: 3.9.4
- PyTorch version (GPU?): 1.13.1.post200 (True)
- Huggingface_hub version: 0.12.0
- Transformers version: 4.25.1
- Accelerate version: 0.15.0
- xFormers version: not installed

**Comment by patrickvonplaten** [comment]
Do you mean models are overfitting? 
Maybe reducing learning rate & getting more data could help?

**Comment by zetyquickly** [comment]
I've noticed huge difference during training Dreambooth with `bnb.Adam8bit` and `deepspeed.ops.adam.DeepSpeedCPUAdam` using default settings. First, one overfits with after 400 steps on 10 images with prior preservation, the second one shows descent results at 800 and won't overfit even at 1600 steps.
